stl/inc/xstring

```diff
@@ -1278,6 +1278,10 @@ public:
 #endif // _CONTAINER_DEBUG_LEVEL > 0
     }
 
+#if _HAS_CXX23
+    constexpr basic_string_view(nullptr_t) noexcept = delete;
+#endif

```

Comment:
I put `noexcept` here to keep the consistency with other `string_view` constructors, should I keep it or remove it?

Reply:
Also missing endif comment

Reply:
I'm fine with removing `noexcept`, since (1) it's not present in the proposal, and (2) it doesn't matter on a deleted function.